log/record_test.go

```diff
@@ -103,3 +103,101 @@ func TestRecordAttributes(t *testing.T) {
 		}
 	})
 }
+
+func BenchmarkRecord(b *testing.B) {
```

Comment:
nit: For consistency I suggest moving this to `record_bench_test.go` or move all benchamarks to `bench_test.go`.